octopod-backend/src/Octopod/Server.hs

```diff
@@ -1045,7 +1045,8 @@ selectDeploymentMetadata conn dName = do
   let
     q =
       "SELECT key, value FROM deployment_metadata \
-      \WHERE deployment_id = (SELECT id FROM deployments WHERE name = ?)"
+      \WHERE deployment_id = (SELECT id FROM deployments WHERE name = ?) \
+      \ORDER BY ord ASC"
```

Comment:
`deployment_metadata` table has `id` field (primary key with auto-increment). Can use it instead of `ord` field.

Reply:
Good point